plugins/postcss-design-tokens/README.md

```diff
@@ -81,6 +81,34 @@ This allows you to generate multiple themed stylesheets.
 
 By default only `@design-tokens` without any `when('foo')` conditions are used.
 
+### functionName
+
+The `functionName` option allows you to set a custom alias for `design-token`.
```

Comment:
Should we anticipate future functions and name this something more specific?

maybe `valueFunctionName`?

Not entirely happy with that but it is the name for the function to replace values.
A bit verbose, but covers it fully : `valueReplacementFunctionName`

Reply:
That makes complete sense!